src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -146,7 +149,7 @@ function RetrieveBannerDynamicAssets() {
   useEffect(() => {
     async function getBackgroundImgSrc() {
       // Uses https://unsplash.com/collections/894/earth-%26-planets collection
-      /* Other Options: 
+      /* Other Options:
```

Comment:
Careful with unrelated changes like this.  It's fine here, but don't change git blame if you don't have to: it makes it harder to understand the history of this file.